starter-analytics-postgres/src/test/groovy/io/micronaut/starter/analytics/postgres/StoreGeneratedProjectStatsSpec.groovy

```diff
@@ -73,8 +75,8 @@ class StoreGeneratedProjectStatsSpec extends Specification implements TestProper
         unauthorizedClient.applicationGenerated(generated).get()
 
         then:
-        Exception e = thrown()
-        e.cause.message == "Unauthorized"
+        ExecutionException e = thrown()
```

Comment:
Can you not use?

```suggestion
        HttpClientResponseException e = thrown()
```

Reply:
Not really. The HTTP client returns a `CompletableFuture` and therefore `HttpClientResponseException` is wrapped inside a `ExecutionException`.